lib/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fs = require('fs');
+const rewriter = require('./rewriter');
+
+/**
+ * @param {string} filePath - the path to the file
+ */
+function rewriteFile(filePath) {
+  const file = {
```

Comment:
Here I feel like the `file` variable doesn't add much, so we can just change it to `rewriter({ ... })`.